lib/Mojo/IRC.pm

```diff
@@ -583,7 +583,7 @@ sub _read {
 
     if ($method =~ /^\d+$/) {
       $self->emit("irc_$method" => $message);
-      $method = IRC::Utils::numeric_to_name($method) or return;
+      $method = IRC::Utils::numeric_to_name($method) or next;
```

Comment:
I think this looks strange: Both your change and the original. Why not have this instead:

    $method = IRC::Utils::numeric_to_name($method) || $method;

Or maybe we should even have support for custom numeric mapping:

    our %NUMERIC_TO_NAME;
    $method = IRC::Utils::numeric_to_name($method) || $NUMERIC_TO_NAME{$method} || do { warn "No name for $method"; $method };

(Need to clean up that last part - just tried to get the idea across)